spec-tests/src/test/java/org/candlepin/spec/imports/ImportUpdateSpecTest.java

```diff
@@ -90,12 +89,7 @@ public void shouldUpdateContentFromManifest() {
 
         // Create a pre-existing content instance that will get overwritten by the content from the manifest
         ContentDTO fromExport = ownerContentApi.getOwnerContent(exportOwnerKey, content.getId());
-        ContentDTO preexistingContent = new ContentDTO().name(fromExport.getName())
-            .id(fromExport.getId())
-            .label(fromExport.getLabel())
-            .type(fromExport.getType())
-            .vendor(fromExport.getVendor())
-            .requiredTags("dummy tags");
+        ContentDTO preexistingContent = Contents.copy(fromExport).requiredTags("dummy tags");
```

Comment:
While these changes are good for long-term maintenance, in this specific case these probably conflict with changes made to main already. I imagine these are worth pulling back on to save some rebase pain -- this entire file got refactored very recently.

Reply:
Done.